src/main/java/uk/gov/ons/ctp/response/action/client/PartySvcClientService.java

```diff
@@ -82,7 +71,7 @@ public PartyDTO getPartyWithAssociationsFilteredBySurvey(
         .debug("Retrieving party");
 
     final MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
-    queryParams.put("survey_id", Arrays.asList(surveyId));
+    queryParams.put("survey_id", Collections.singletonList(surveyId));
```

Comment:
why isn't this just the surveyId? There's only ever going to be one surveyId!

Reply:
I'm assuming it's because of what format the party endpoint is expecting it to be sent in but I'll test if things work if it's not in a list